runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -411,6 +420,26 @@ else if (expeditiousCheckMatcher.find())
 			{
 				updateExpeditiousBraceletCharges(Integer.parseInt(expeditiousCheckMatcher.group(1)));
 			}
+			else if (faladorShieldCheckMatcher.find())
+			{
+				final String match = faladorShieldCheckMatcher.group(1);
+				if (match.equals("one"))
+				{
+					updateFaladorShieldCharges(1);
+				}
+				else if (match.equals("two"))
+				{
+					updateFaladorShieldCharges(2);
+				}
+			}
+			else if (message.equals(FALADOR_SHIELD_EMPTY_TEXT))
+			{
+				updateFaladorShieldCharges(0);
+			}
+			else if (message.equals(FALADOR_SHIELD_EMPTY_TEXT_RECHARGE))
+			{
+				updateFaladorShieldCharges(0);
+			}
```

Comment:
These messages both yield the same code path; combine the conditions instead of duplicating the code body for each of them.
```suggestion
			else if (message.equals(FALADOR_SHIELD_EMPTY_TEXT) || message.equals(FALADOR_SHIELD_EMPTY_TEXT_RECHARGE))
			{
				updateFaladorShieldCharges(0);
			}
```